### ISSUE TITLE
Jesus sent me here.

### ISSUE THREAD

**Issue by BlackShadow** [luzeadev/winactivate#26]
![105945620 465_image](https://user-images.githubusercontent.com/30868395/148656614-fba68d11-77b2-4fd2-9911-4f7c641c28e5.png)

**Comment by j4ck-code** [comment]
Yeah, me too, twitter people kinda don't know shit about technology so this is the time that they're 10% correct.
they gave the link to ADOBE SUITE 2022. 2022. BRO DON'T USE THAT

**Comment by dotada** [comment]
I was sent here by jesus aswell.
